CHANGELOG.md

```diff
@@ -6,6 +6,17 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [14.0.0]
+### Changed
+- **BREAKING:** Adapt to EIP-1193 provider changes by deprecating the `sendAsync` method and replacing with the `request` method ([#317](https://github.com/MetaMask/eth-json-rpc-middleware/pull/317))
+  - **BREAKING:** Refactor `providerAsMiddleware` and middleware functions `retryOnEmpty`, `block-ref` to use the `sendAsync` method.
```

Comment:
Thanks for pointing that out @legobeat, I upgraded this to a major release.

@cryptodev-2s Could you take a look at the changelog entries for the EIP-1193 update? Feel free to make edits.

Reply:
@MajorLift what has been done here actually is not deprecating `sendAsync` it's already declared as deprecated with the new version of Safe Event Emitter Provider that has been bumped  the main work that has been done here is replacing the usage of `sendAsync` with `request`.

Reply:
```suggestion
- **BREAKING:** Refactor `providerAsMiddleware` and middleware functions `retryOnEmpty`, `block-ref` to use the `request` method.
```

Reply:
@legobeat The `publish-release` action is triggered by the commit being pushed onto main, at which point the release candidate branch has been deleted. It also uses package.json to pull the `RELEASE_VERSION`, so I'm pretty sure this shouldn't be an issue.

Either way, I'd like to try merging this with the current branch name just to be sure for future reference.🧐